src/server/scripts/Outland/BlackTemple/boss_illidari_council.cpp

```diff
@@ -0,0 +1,926 @@
+        {
+            DoCastSelf(SPELL_SEAL_OF_BLOOD);
+            events.ScheduleEvent(EVENT_BLESS, Seconds(20));
+            events.ScheduleEvent(EVENT_CONSECRATION, Seconds(10));
+            events.ScheduleEvent(EVENT_HAMMER_OF_JUSTICE, Seconds(10));
+            events.ScheduleEvent(EVENT_JUDGEMENT, Seconds(15));
+            events.ScheduleEvent(EVENT_AURA, Seconds(6));
+        }
+
+        void ExecuteEvent(uint32 eventId) override
+        {
+            switch (eventId)
+            {
+                case EVENT_BLESS:
+                {
+                    std::list<Unit*> TargetList;
+                    Trinity::AnyFriendlyUnitInObjectRangeCheck checker(me, me, 100.0f);
+                    Trinity::UnitListSearcher<Trinity::AnyFriendlyUnitInObjectRangeCheck> searcher(me, TargetList, checker);
+                    me->VisitNearbyObject(100.0f, searcher);
+
+                    if (!TargetList.empty())
+                    {
+                        Unit* target = Trinity::Containers::SelectRandomContainerElement(TargetList);
+                        DoCast(target, RAND(SPELL_BLESS_PROTECTION, SPELL_BLESS_SPELL_WARDING));
+                    }
+                    events.Repeat(Seconds(30), Seconds(45));
+                    break;
+                }
+                case EVENT_AURA:
+                    DoCastSelf(RAND(SPELL_CHROMATIC_AURA, SPELL_DEVOTION_AURA));
+                    events.Repeat(Seconds(30));
+                    break;
+                case EVENT_HAMMER_OF_JUSTICE:
+                    if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM, 1, HammerTargetSelector(me)))
+                        DoCast(target, SPELL_HAMMER_OF_JUSTICE);
+                    events.Repeat(Seconds(20));
+                    break;
+                case EVENT_JUDGEMENT:
+                    DoCastVictim(SPELL_JUDGEMENT);
+                    events.Repeat(Seconds(15));
+                    break;
+                case EVENT_CONSECRATION:
+                    DoCastSelf(SPELL_CONSECRATION);
+                    events.Repeat(Seconds(30), Seconds(35));
+                    break;
+                default:
+                    break;
+            }
+        }
+    };
+
+    CreatureAI* GetAI(Creature* creature) const override
+    {
+        return GetBlackTempleAI<boss_gathios_the_shattererAI>(creature);
+    }
+};
+
+class boss_high_nethermancer_zerevor : public CreatureScript
+{
+public:
+    boss_high_nethermancer_zerevor() : CreatureScript("boss_high_nethermancer_zerevor") { }
+
+    struct boss_high_nethermancer_zerevorAI : public IllidariCouncilBossAI
+    {
+        boss_high_nethermancer_zerevorAI(Creature* creature) : IllidariCouncilBossAI(creature, DATA_HIGH_NETHERMANCER_ZEREVOR), _canUseArcaneExplosion(true) { }
+
+        void Reset() override
+        {
+            IllidariCouncilBossAI::Reset();
+            _canUseArcaneExplosion = true;
+            DoCastSelf(SPELL_DAMPEN_MAGIC);
```

Comment:
Done.
He cast before start fight (on reset) and cast again when combat start. After debuff expire or dispel, he cast again after 50 sec